.github/actions/bitcoin-int-tests/Dockerfile.bitcoin-tests

```diff
@@ -12,4 +12,14 @@ RUN cd / && tar -xvzf bitcoin-0.20.0-x86_64-linux-gnu.tar.gz
 RUN ln -s /bitcoin-0.20.0/bin/bitcoind /bin/
 
 ENV BITCOIND_TEST 1
-RUN cd testnet/stacks-node && cargo test -- --test-threads 1 --ignored
+WORKDIR /src/testnet/stacks-node
+RUN cargo test -- --test-threads 1 --ignored tests::integrations::integration_test_get_info
+RUN cargo test -- --test-threads 1 --ignored tests::neon_integrations::bitcoind_integration_test
+RUN cargo test -- --test-threads 1 --ignored tests::neon_integrations::size_check_integration_test
+RUN cargo test -- --test-threads 1 --ignored tests::neon_integrations::liquid_ustx_integration
+RUN cargo test -- --test-threads 1 --ignored tests::neon_integrations::stx_transfer_btc_integration_test
+RUN cargo test -- --test-threads 1 --ignored tests::neon_integrations::bitcoind_forking_test
+RUN cargo test -- --test-threads 1 --ignored tests::neon_integrations::microblock_integration_test
+RUN cargo test -- --test-threads 1 --ignored tests::neon_integrations::pox_integration_test
+RUN cargo test -- --test-threads 1 --ignored tests::bitcoin_regtest::bitcoind_integration_test
+RUN cargo test -- --test-threads 1 --ignored tests::should_succeed_handling_malformed_and_valid_txs
```

Comment:
Is there a way to programmatically enumerate this list, so we don't accidentally forget to add new ones to the test runner as they get created?  Same question for Atlas tests.

Reply:
No, I don't believe so